x-pack/legacy/plugins/siem/public/components/paginated_table/index.tsx

```diff
@@ -0,0 +1,315 @@
+                position="absolute"
+                zIndex={3}
+                data-test-subj="LoadingPanelPaginatedTable"
+              />
+            </>
+          )}
+
+          <HeaderPanel
+            id={id}
+            showInspect={showInspect}
+            subtitle={`${i18n.SHOWING}: ${headerCount.toLocaleString()} ${headerUnit}`}
+            title={headerTitle}
+            tooltip={headerTooltip}
+          >
+            {headerSupplement}
+          </HeaderPanel>
+
+          <BasicTable
+            items={pageOfItems}
+            columns={columns}
+            onChange={onChange}
+            sorting={
+              sorting
+                ? {
+                    sort: {
+                      field: sorting.field,
+                      direction: sorting.direction,
+                    },
+                  }
+                : null
+            }
+          />
+          <FooterAction>
+            <EuiFlexGroup alignItems="center">
+              <EuiFlexItem>
+                {!isEmpty(itemsPerRow) && (
+                  <EuiPopover
+                    id="customizablePagination"
+                    data-test-subj="loadingMoreSizeRowPopover"
+                    button={button}
+                    isOpen={isPopoverOpen}
+                    closePopover={closePopover}
+                    panelPaddingSize="none"
+                  >
+                    <EuiContextMenuPanel items={rowItems} data-test-subj="loadingMorePickSizeRow" />
+                  </EuiPopover>
+                )}
+              </EuiFlexItem>
+              <PaginationWrapper grow={false}>
+                <EuiPagination
+                  data-test-subj="numberedPagination"
+                  pageCount={pageCount}
+                  activePage={activePage}
+                  onPageClick={goToPage}
+                />
+              </PaginationWrapper>
+            </EuiFlexGroup>
+          </FooterAction>
+        </BasicTableContainer>
+      </EuiPanel>
+    );
+  }
+);
+
+export const BasicTableContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  height: auto;
+  position: relative;
+`;
+
+const FooterAction = styled.div`
+  margin-top: 0.5rem;
+  width: 100%;
+`;
+
+/*
+ *   The getOr is just there to simplify the test
+ *   So we do NOT need to wrap it around TestProvider
+ */
+const BackgroundRefetch = styled.div`
+  background-color: ${props => getOr('#ffffff', 'theme.eui.euiColorLightShade', props)};
+  margin: -5px;
+  height: calc(100% + 10px);
+  opacity: 0.7;
+  width: calc(100% + 10px);
+  position: absolute;
+  z-index: 3;
+  border-radius: 5px;
+`;
+
+const BasicTable = styled(EuiBasicTable)`
+  tbody {
+    th,
+    td {
+      vertical-align: top;
+    }
+  }
+`;
+
+const PaginationEuiFlexItem = styled(EuiFlexItem)`
+  button.euiButtonIcon.euiButtonIcon--text {
+    margin-left: 20px;
+  }
+  .euiPagination::before {
+    content: '\\2026';
```

Comment:
Minor nit, but this should only require one `\`. So `\2026` is all you need.

Reply:
nope, this is JS so it reads it as an escaped character. it doesnt work without it

Reply:
Ahhhh, my bad. Still getting used to CSS in JS.